string goal
int16 timeStep
---
Action parsed